python/modem-firmware-1.3+/README.md

```diff
@@ -67,245 +67,29 @@ If the `rtt` option is specified, communication will be performed using [SEGGER'
 To use RTT, the device must be running the [Modem Shell](https://github.com/nrfconnect/sdk-nrf/tree/main/samples/nrf9160/modem_shell) sample application [built with the RTT overlay](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/modem_shell/README.html#segger-rtt-support).
 This script will optionally flash the modem shell application on startup if a hex file path is provided with the `mosh_rtt_hex` option.
 
-```
```

Comment:
I notice the other scripts show the help but you removed all of the help from this one.

I would like the help back.

I'm fine with the removal of the verbose examples from this.